ndcube/wcs/wrappers/tests/test_resampled_wcs.py

```diff
@@ -195,22 +195,21 @@ def nine_nine_wcs():
     wcs.wcs.crpix = [1, 1]
 
     wcs.wcs.set()
-    wcs.pixel_shape = [9, 9]
+    wcs.pixel_shape = [4, 5]
     return wcs
 
-def test_corner_pixels(nine_nine_wcs):
-    owcs = nine_nine_wcs
-    # original corner
-    ocorner = owcs.pixel_to_world_values(-0.5, -0.5)
-    # one super pixel along (3 original pixels)
-    osone = owcs.pixel_to_world_values(2.5, 2.5)
-
-    wcs = ResampledLowLevelWCS(owcs, 3)
-
-    # resampled corner == original corner
-    corner = wcs.pixel_to_world_values(-0.5, -0.5)
-    # One super pixel along
-    sone = wcs.pixel_to_world_values(0.5, 0.5)
-
-    assert np.allclose(ocorner, corner)
-    assert np.allclose(osone, sone)
+@pytest.mark.parametrize(('factor', 'offset', 'expected_over_pixels'),
+                         [([2, 3], [0, 0], np.meshgrid(np.arange(-0.5, 1.75, 0.25), np.arange(-0.5, 1.2, 1/6))),
+                          ([2, 3], [1, 2], np.meshgrid(np.arange(-1, 1.25, 0.25), np.arange(-7, 4) / 6)),
```

Comment:
Can you clearly explain these magic numbers, this makes it really hard to understand what the test is actually doing.